packages/smikims-arpspoof/PKGBUILD

```
@@ -33,6 +33,6 @@ package() {
   install -Dm 755 arpspoof "$pkgdir/usr/bin/$pkgname"
   install -m 755 sweep.sh "$pkgdir/usr/bin/$pkgname-sweep"
   install -m 755 cage.sh "$pkgdir/usr/bin/$pkgname-cage"
-  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/README.md"
```

Shouldn't it rather be?

```suggestion
  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
```

Similar to https://github.com/BlackArch/blackarch-pkgbuilds/blob/d44f5dee7e2f20161bdb10199cce7f5a322f4e38/PKGBUILD-generic-jointscript#L37, deploy a file in a folder.

---

Yep, ur right